install/share/indices.ldif

```diff
@@ -271,6 +305,22 @@ nsSystemIndex: false
 nsIndexType: eq
 nsIndexType: pres
 
+dn: cn=ipServicePort,cn=index,cn=userRoot,cn=ldbm database,cn=plugins,cn=config
+changetype: add
+cn: ipServicePort
+objectClass:top
+objectClass:nsIndex
+nsSystemIndex: false
+nsIndexType: eq
+
+dn: cn=ipServiceProtocol,cn=index,cn=userRoot,cn=ldbm database,cn=plugins,cn=config
+changetype: add
+cn: ipServiceProtocol
+objectClass:top
+objectClass:nsIndex
+nsSystemIndex: false
+nsIndexType: eq
```

Comment:
IPA doesn't provide either ipServicePort or ipServiceProtocol as far as I can find. So why add an index for attributes that don't exist (by default anyway)?

Reply:
389-DS doesn't know that IPA doesn't have any entries with these attributes. When SSSD's nsswitch plugin looks for services, the LDAP server has to scan the database.

Example from https://pagure.io/freeipa/issue/7786

```
  Unindexed Component #2 (notes=U)
  -  Date/Time:             14/Nov/2018:18:31:11
  -  Connection Number:     3866581
  -  Operation Number:      27
  -  Etime:                 1.1235918834
  -  Nentries:              0
  -  IP Address:            Unknown_Host
  -  Search Base:           cn=accounts,dc=example,dc=local
  -  Search Scope:          2 (subtree)
  -  Search Filter:         (&(ipserviceport=57672)(ipserviceprotocol=tcp)(objectclass=ipservice))
```

It took over 1.1 seconds to figure out that there is no entry with an ipServicePort and ipServiceProtocol.

Reply:
Right so we should ask SSSD to stop doing this, or explain why they are.

Reply:
I also wonder if the order of filters matters, too. Perhaps ``(&(objectclass=ipservice)(ipserviceport=57672)(ipserviceprotocol=tcp))`` is faster than `` (&(ipserviceport=57672)(ipserviceprotocol=tcp)(objectclass=ipservice))``. I know some SQL servers have a query optimizer that reorder the terms to benefits from index with better selectability.

Reply:
Also a subtree query on ``cn=accounts,dc=example,dc=local`` is pretty costly. The accounts subtree can easily contain tenths of thousands of entries.

Reply:
The ipService performance issue is now tracked in ticket https://pagure.io/freeipa/issue/7797 . I've removed the indices from this PR.